projects/hslayers/components/wfs-filter/wfs-filter.component.ts

```diff
@@ -0,0 +1,325 @@
+import * as olFormatFilter from 'ol/format/filter';
+import {AsyncPipe, NgClass} from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Signal,
+  inject,
+} from '@angular/core';
+import {FormsModule} from '@angular/forms';
+import {HsEventBusService} from 'hslayers-ng/services/event-bus';
+import {HsFiltersComponent, HsFiltersService} from 'hslayers-ng/common/filters';
+import {HsLayerDescriptor, WfsFeatureAttribute} from 'hslayers-ng/types';
+import {
+  HsLayerManagerService,
+  HsLayerSelectorService,
+} from 'hslayers-ng/services/layer-manager';
+import {HsLayoutService} from 'hslayers-ng/services/layout';
+import {
+  HsPanelBaseComponent,
+  HsPanelHeaderComponent,
+} from 'hslayers-ng/common/panels';
+import {HsUtilsService} from 'hslayers-ng/services/utils';
+import {HttpClient} from '@angular/common/http';
+import {TranslateCustomPipe} from 'hslayers-ng/services/language';
+import {Vector as VectorLayer} from 'ol/layer';
+import {Vector as VectorSource} from 'ol/source';
+import {catchError, filter, lastValueFrom, map, switchMap, tap} from 'rxjs';
+import {computed, signal} from '@angular/core';
+import {
+  getWfsAttributes,
+  getWfsUrl,
+  setWfsAttributes,
+} from 'hslayers-ng/common/extensions';
+import {takeUntilDestroyed, toSignal} from '@angular/core/rxjs-interop';
+
+@Component({
+  selector: 'hs-wfs-filter',
+  templateUrl: './wfs-filter.component.html',
+  styles: `
+    .hs-wfs-filter-panel > div {
+      padding: 0 0.75rem;
+    }
+  `,
+  standalone: true,
+  imports: [
+    NgClass,
+    HsFiltersComponent,
+    FormsModule,
+    AsyncPipe,
+    HsPanelHeaderComponent,
+    TranslateCustomPipe,
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class HsWfsFilterComponent extends HsPanelBaseComponent {
+  name = 'wfsFilter';
```

Comment:
consider:

* Rename this new panel to just "filter", because it already exists as a value in `HsConfig.panelsEnabled` (kind of a similar panel was already implemented years ago and this param is a relict).
* Or remove the value "filter" from panelsEnabled for good.
* Or work with both values as with synonyms.

I have a weak preference for (1), but the two other options are also fine.

Reply:
I'd like to keep 'wfs' in the name to communicate the  functionality properly. 
Regarding the `panelsEnabled.filter` I think it should be removed - panel no longer exists so this is no brainer for me.

So basically (2) :D